lib/quest/api.rb

```diff
@@ -28,40 +28,40 @@ def active_quest_status
     end
 
     get '/status/examples' do
-      active_quest_status[:examples].to_json
+      active_quest_status['examples'].to_json
     end
 
     get '/status/examples/:number' do
-      active_quest_status[:examples][params[:number].to_i - 1].to_json
+      active_quest_status['examples'][params[:number].to_i - 1].to_json
```

Comment:
The array indexing mismatch is because I want the api format to match with the task numbering in the content, which starts at one.

The hash indexing mismatch is there because parsing RSpec's json output results in a hash with string keys, while Sinatra's params hash uses symbols as keys. It's a little confusing to read, but I don't know if symbolizing or stringifying the keys in one hash to match the other would be an improvement.